src/test/resources/expected.tsv

```diff
@@ -0,0 +1,7 @@
+TEST1	AD	Mental_retardation__autosomal_dominant_42:AD
+TEST2	AD	Mental_retardation__autosomal_dominant_42:AD
+TEST3	AD/SM	{Epilepsy__juvenile_myoclonic__susceptibility_to}:AD/SM,{Epilepsy__idiopathic_generalized__10}:AD,{Epilepsy__generalized__with_febrile_seizures_plus__type_5__susceptibility_to}:AD
+TEST5	AD/SM	{Epilepsy__juvenile_myoclonic__susceptibility_to}:AD/SM,{Epilepsy__idiopathic_generalized__10}:AD,{Epilepsy__generalized__with_febrile_seizures_plus__type_5__susceptibility_to}:AD
+TEST8	AD/SM	{Epilepsy__juvenile_myoclonic__susceptibility_to}:AD/SM,{Epilepsy__idiopathic_generalized__10}:AD,{Epilepsy__generalized__with_febrile_seizures_plus__type_5__susceptibility_to}:AD
+TEST4	AR	?Usher_syndrome__type_1M:AR,Deafness__autosomal_recessive_36:AR,Deafness__neurosensory__without_vestibular_involvement__autosomal_dominant:AR
```

Comment:
is the `?` intended?

Reply:
yes, this occurs in omim, so I want to test if the regular expression can cope with them